src/main/resources/org/sagebionetworks/web/client/widget/profile/UserProfileEditorWidgetViewImpl.ui.xml

```diff
@@ -162,7 +162,7 @@
                 addStyleNames="displayInlineBlock"
                 ui:field="changePasswordLink"
                 target="_blank"
-                href="/Profile:v/settings"
+                href="https://accounts.sagebionetworks.synapse.org/authenticated/myaccount#ChangePassword"
```

Comment:
Link directly to the ChangePassword section on OneSage (see https://github.com/Sage-Bionetworks/synapse-web-monorepo/pull/954)